### ISSUE TITLE
Apple M1

### ISSUE THREAD

**Issue by stefansundin** [andrewn/brew-services-menubar#19]
On Apple M1 computers, Homebrew changed its path (see https://docs.brew.sh/Installation).

Therefore, this program just says "No services available" and doesn't work out of the box.

To fix it you can run:

```
defaults write andrewnicolaou.BrewServicesMenubar brewExecutable /opt/homebrew/bin/brew
```

The code should be updated to detect this case. I was thinking about doing it but I would have to install Xcode and I am not sure I want to do that just yet.

**Comment by andrewn** [comment]
It's a bit more complex than that as you can install Homebrew using Rosetta 2 which means it will still be installed in `/usr/local/bin/brew`.

So I guess the app should try both on start-up and pick the path one that's installed?

I wonder if it's possible to get access to the shell environment so it can use whatever the user's already configured?

**Comment by stefansundin** [comment]
Yeah. I guess if the configuration isn't set, then it should check if one of the files exists and then check the second location.

And I think instead of saying "No services available" it can say "Homebrew not found" if the file isn't there.

Do you think you will have a chance to look at this or should I install Xcode and give it a try myself? It would be nice if #14 could be merged too.

I think it's probably very easy to make a universal package as well, that includes an arm64 binary.

**Comment by andrewn** [comment]
> Yeah. I guess if the configuration isn't set, then it should check if one of the files exists and then check the second location.
> 
> And I think instead of saying "No services available" it can say "Homebrew not found" if the file isn't there.

That sounds perfect.

> 
> Do you think you will have a chance to look at this or should I install Xcode and give it a try myself? It would be nice if #14 could be merged too.
> 
> I think it's probably very easy to make a universal package as well, that includes an arm64 binary.

I won't have time to look at this 😢  If you could open a PR then that would be amazing!

**Comment by andrewn** [comment]
Fixed by #14
